scripts/run_locally.sh

```diff
@@ -3,7 +3,9 @@
 set -e
 
 echo "==> $(date +%H:%M:%S) ==> Starting up environment containers..."
-docker compose up -d \
+docker compose pull \
+  && docker compose down -v \
```

Comment:
Should we add if they want to proceed? for previous run this will remove all the database stored.

Reply:
It will not. Database is stored on `data` folder, is not a named volume